### ISSUE TITLE
Fxxking this Hypocritical and disgusting American company --- Greetings from Syria

### ISSUE THREAD

**Issue by chunk425015** [facebookresearch/detectron2#4017]
All in MMDetection plz.

**Comment by zhaoshishuang** [comment]
🐮🍺

**Comment by spytensor** [comment]
🐮🍺

**Comment by zhanghang1989** [comment]
Thanks for the greetings!
Detectron2 is an open source toolkit which is beneficial to and contributed by all AI communities. We hope that our forum can stay away from politics and discuss research problems.

**Comment by zqxiang** [comment]
> Thanks for the greetings! Detectron2 is an open source toolkit which is beneficial to and contributed by AI community. We hope that our forum can stay away from politics, and only discuss research & engineering problems.

Is "stay away from politics" means "Support Ukraine"？
Seems that my English teacher doesn't understand English.
![image](https://user-images.githubusercontent.com/25455499/156819696-2af1ddc2-fd0b-41b7-86b7-2bef27d4520e.png)

**Comment by chunk425015** [comment]
@zhanghang1989
I agree with you that this community should stay away from politics. But why did you guys approve the PR about adding the `support ukraine` mark? You should stand with the people who suffer from the war instead of Ukraine. 
Many people from all around the world have contributed to this repo, you guys have no right to add such a biased political slogan on behave of all the contributors.
It's very stupid to approve the PR. Detectron2 is a well-known codebase in the AI community, and this community should not be used as a tool to publicize personal political views.

Dr Zhang, I am also a member of the AI community and have read some of your work. This response is not a personal attack. I can not stand with hypocritical and disgusting behavior representing the interests of NATO. So I suggest you do not involve in this and urge your colleagues to revert this PR as nodejs did early days. It is also a good idea to the button to `support suffering people`.

And I also wish to keep this issue open for community discussion.
